lib/indexers/projects-content/extract-content.js

```diff
@@ -0,0 +1,139 @@
+require('@asl/service/lib/register');
```

Comment:
What does this do?

Reply:
It includes a pre-configured implementation of babel-register which rewires `require` to also do a babel transpile on any files it `require`s.

Specifically in this case it allows us to load the schema files from `@asl/projects` without getting parsing errors when node finds the `import` statements in those files.

Reply:
👍